### DISCUSSION TITLE
Không thể kết nối đến máy chủ EVN

### DISCUSSION THREAD

**Discussion by leolionart** [trvqhuy/ha-evn#40] [General]
ERROR (MainThread) [custom_components.nestup_evn.nestup_evn] Cannot connect to EVN Server while requesting new data: status code 500

**Comment by Anhnq09** [comment]
Unexpected exception: Cannot connect to host cskh.evnhcmc.vn:443 ssl:<ssl.SSLContext object at 0x7f8ffff6d0> [[SSL: UNSAFE_LEGACY_RENEGOTIATION_DISABLED] unsafe legacy renegotiation disabled (_ssl.c:1000)]
